spacy/tokens/doc.pyx

```diff
@@ -820,6 +820,69 @@ cdef class Doc:
             set_children_from_heads(self.c, length)
         return self
 
+    @staticmethod
+    def from_docs(docs, space_delimiter=True, attributes=None):
```

Comment:
ok, I saw the parameter name in some other methods like `merge` or `bulk_merge`, but this is because it refers to a `dict` instead of a `list` I guess. Changed it.